devel/ldid/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           xcode 1.0
+
+name                ldid
+version             2.1.2
+set git_commit      c2f8abf013b22c335f44241a6a552a7767e73419
+categories          devel
+platforms           darwin
+maintainers         {@i0ntempest me.com:szf1234} openmaintainer
+license             GPL-3
+
+description         Link Identity Editor
+long_description    Link Identity Editor lets you manipulate the signature block in a Mach-O binary.
+
+homepage            https://cydia.saurik.com/info/ldid/
+
+depends_lib-append  port:libxml2 \
+                    path:lib/libssl.dylib:openssl
+
+fetch.type          git
+git.url             --recursive --depth 2000 https://git.saurik.com/ldid.git
```

Comment:
Why is `--depth 2000` necessary?

I'm not sure `git.url` should be used to pass extra options to `git clone`, even if it works; this is not something any other ports appear to do. A few ports either define their own fetch or post-fetch phases if they need more control over `git clone`. But maybe this indicates a lack of `git.*` options to do so.

Reply:
I copied the command from another portfile so I thought it’s needed. I’ll change it later to use the documented way for dealing with submodules (post-fetch).